lib/css/components/post-summary.less

```diff
@@ -403,10 +403,9 @@
 
 .s-post-summary__ignored,
 .s-post-summary__deleted {
-    .s-post-summary--content > *:not(.s-post-summary--content-menu-button) {
+    .s-post-summary--content > *:not(.s-post-summary--content-menu-button):not(.s-post-summary--meta) {
```

Comment:
I see a future where this has like 10 `:not()`s. That's:not(good).

I've captured this issue in my audit doc.